src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -272,6 +295,9 @@ private async Task UpdateAdvertisingManifestAsync(WorkloadManifestInfo manifest,
             var manifestId = new ManifestId(manifest.Id);
             string currentFeatureBand = _sdkFeatureBand.ToString();
 
+
+            // TODO: abstract the fallback logic so it can be used in the get manifest package downloads async
```

Comment:
Is the TODO necessary or related to a separate issue?

Reply:
this TODO is left over from earlier in my work. It's relevant anymore